web/src/components/EventGalleryComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Masonry from "react-masonry-css";
+import { EventGallery } from "../types/types";
+
+interface EventGalleryProps {
+  photos: EventGallery[];
+}
+
+const EventGalleryComponent: React.FC<EventGalleryProps> = ({ photos }) => {
+  const breakpointColumnsObj = {
+    default: 3,
```

Comment:
Im not familiar with this library, Is this in "px" if so, is it possible to change to "rem". If not it's fine dw about it

Reply:
Nah, it means that it will display 3 images per row. and the breakpoints are there for when the screen size changes - so for example -> when screen size 1100 pixels or more then 3 images, if less than 700, then 2 images per row etc!